change/@fluentui-react-charting-4b4305e3-4ff3-49f0-88b1-d147b5063a05.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Set axisType for single data point",
```

Comment:
>point [](http://example.com/codeflow?start=43&length=5)

mention that this change is for areachart

Reply:
Ok

Reply:
Fixed